jobs/JREGIONAL_NOVARCLD

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
A picky comment on the JJOB name, would `NONVAR` reads more clear than `NOVAR`. The latter sounds a little bit like "don't do var cloud anx".  If so, would need to change all "novar" to "nonvar" in all related files.

Reply:
Changed.